arches/app/templates/views/components/widgets/resource-instance-select.htm

```diff
@@ -54,17 +54,17 @@
                 <div class='rr-table-row'>
                     <div class="rr-table-row-initial">
                         <div class='rr-table-column icon-column'>
-                            <button data-bind="click: function(){window.open($parent.resourceEditorUrl+ko.unwrap($data.resourceId))}, clickBubble: false">
+                            <button data-bind="click: function(){window.open($parent.resourceEditorUrl+ko.unwrap($data.resourceId))}, clickBubble: false" aria-label="Edit Resource Instance">
                                 <i class="fa fa-pencil"></i>
                             </button>
                         </div>
                         <div class='rr-table-column icon-column'>
-                            <button data-bind="click: $parent.deleteRelationship, clickBubble: false">
+                            <button data-bind="click: $parent.deleteRelationship, clickBubble: false" aria-label="Delete Resource Instance">
```

Comment:
This won't delete the resource instance, it will remove the resource instance relationship.

Reply:
Should be fixed now